Modules/Core/QuadEdgeMesh/test/itkQuadEdgeMeshPolygonCellTest.cxx

```diff
@@ -86,6 +87,20 @@ itkQuadEdgeMeshPolygonCellTest(int, char *[])
    */
   mesh->SetCellsAllocationMethod(itk::MeshEnums::MeshClassCellsAllocationMethod::CellsAllocatedDynamicallyCellByCell);
 
+
+  // Test point container iterators
+  auto * quadEdgeMeshPolygonCell = new PolygonCellType();
+
+  PolygonCellType::PointIdIterator pointId = quadEdgeMeshPolygonCell->PointIdsBegin();
+  PolygonCellType::PointIdIterator endId = quadEdgeMeshPolygonCell->PointIdsEnd();
+  ITK_TEST_EXPECT_TRUE(pointId);
```

Comment:
The `PointIdsBegin` method creates point ids, even if the point container is empty:
https://github.com/InsightSoftwareConsortium/ITK/blob/89d3c3c0158806ab44a33d3c76b436537864398e/Modules/Core/QuadEdgeMesh/include/itkQuadEdgeMeshPolygonCell.h#L202

Not sure about that behavior. Also, the Ids created seem to be random.

Reply:
If you don't plan to look into this now/soon, perhaps add it to the issue tracker?

Reply:
Do not really plan to track this down at this time. Will open an issue.

Reply:
Opened #4158.